index.html

```diff
@@ -51,5 +51,154 @@ <h1>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          <ul class="tag">
+            <li class="html">html</li>
+            <li class="cascade">css</li>
+            <li class="javascript">javascript</li>
+          </ul>
+
+          <div class="see-project-one">
+           <button class="see-project">See project</button>
+           </div>
+
+      </section>
+
+      <section class="card-Work-container-2">
+        <div class="snapshot-porfolio">
+          <img src="./images/snap 2.png" alt="">
+        </div>
+        <h2>Multi-Post <br> Stories</h2>
+        <ul class="tonic">
+          <li class="canopy">CANOPY</li>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          <ul class="tag">
+            <li class="html">html</li>
+            <li class="cascade">css</li>
+            <li class="javascript">javascript</li>
+          </ul>
+          <div class="see-project-one">
+           <button class="see-project">See project</button>
+           </div>
+
+      </section>
+
+      <section class="card-Work-container">
+        <div class="snapshot-porfolio">
+          <img src="./images/snap 3.png" alt="">
+        </div>
+        <h2>Tonic</h2>
+        <ul class="tonic">
+          <li class="canopy">CANOPY</li>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+      
+        <div>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          </div>
+          <ul class="tag">
+            <li class="html">html</li>
+            <li class="cascade">css</li>
+            <li class="javascript">javascript</li>
+          </ul>
+
+          <div class="see-project-one">
+           <button class="see-project">See project</button>
+           </div>
+
+      </section>
+
+      <section class="card-Work-container-2">
+        <div class="snapshot-porfolio">
+          <img src="./images/snap 4.png" alt="">
+        </div>
+        <h2>Multi-Post <br> Stories</h2>
+        <ul class="tonic">
+          <li class="canopy">CANOPY</li>
+          <li class="back-end">Back End Dev</li>
+          <li class="year">2015</li>
+        </ul>
+        <p>A daily selection of privately 
+          personalized reads; no accounts 
+          or sign-ups required.</p>
+          <ul class="tag">
+
+              <li class="html">html</li>
+              <li class="cascade">css</li>
+              <li class="javascript">javascript</li>
+            </ul>
+          <div class="see-project-one">
+           <button class="see-project">See project</button>
```

Comment:
- [ ] Please Implement the hover effects on all of your _see project buttons_, this will improve the UI of our site see the figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/201485970-c9b58ad0-ad60-4b3b-af7c-fad137c756f6.png)